app/App.js

```diff
@@ -9,7 +8,7 @@ export default class App extends React.Component {
 
   render() {
     return (
-      <div className="sn-component windows-web">
```

Comment:
`windows-web` was added on https://github.com/standardnotes/simple-task-editor/commit/99ddce7bb31a8e006633b2c185296e57a8d5dd4b#diff-927334fc49930ff6860b35d0bc3943fc84d050b4f6b3b3b8e1d15a1e9c90f740

`windows-web` shares the same styles as `windows-desktop`, `linux-web` and `linux-desktop`. I'm assuming this is a fix for MacOS, correct?

Reply:
Hmm...I don't recall but this seems like testing code that should have been deleted. Let's just ensure cross-editor consistency. So if none of the other editors do this, this one shouldn't either.